content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
-from the purchase order form, then click :guilabel:`Create Alternative`. When clicked, a
-:guilabel:`Create alternative` pop-up window appears.
+To create alternative |RfQs|, from the |PO| form, click the :guilabel:`Alternatives` tab. Then,
+click :guilabel:`Create Alternative`. When clicked, a :guilabel:`Create alternative` pop-up window
+appears.
 
-.. image:: calls_for_tenders/calls-for-tenders-alternatives-popup.png
+.. image:: calls_for_tenders/calls-for-tenders-create-alternative.png
    :align: center
    :alt: Calls for tenders pop-up to create alternative quotation.
 
-From this window, select a new/different vendor from the drop-down menu next to the
-:guilabel:`Vendor` field to assign this alternative quotation to.
+From this window, select an alternative vendor from the drop-down menu next to the
+:guilabel:`Vendor` field, to whom the alternative quotation is assigned.
 
 Next to this, there is a :guilabel:`Copy Products` checkbox that is selected by default. When
-selected, the product quantities of the original :abbr:`PO (Purchase Order)` are copied to the
-alternative. For this first alternative quotation, leave the checkbox checked. Once finished, click
-:guilabel:`Create Alternative`. This creates (and navigates to) a new :abbr:`PO (Purchase Order)`.
+selected, the product quantities of the original |PO| are copied over to the alternative. For this
+first alternative quotation, leave the checkbox checked. Once finished, click :guilabel:`Create
+Alternative`. This opens a new |PO|.
 
-Since the :guilabel:`Create Alternative` checkbox was left checked, this new purchase order form is
-already populated with the same products, quantities, and other details as the previous, original
-:abbr:`PO (Purchase Order)`.
+Since the :guilabel:`Create Alternative` checkbox was left checked, the new form is already
+pre-populated with the same products, quantities, and other details as the previous, original |PO|.
 
 .. note::
    When the :guilabel:`Copy Products` checkbox is selected while creating an alternative quotation,
-   additional products do not need to be added on the purchase order form unless desired. However,
-   if a chosen vendor is listed in the :guilabel:`Vendor` column under the :guilabel:`Purchase` tab
-   on a product form included in the purchase order, the values set on the product form carry over
-   to the :abbr:`PO (Purchase Order)`, and have to be changed manually, if desired.
+   additional products do **not** need to be added unless desired.
+
+   However, if a chosen vendor is listed in the :guilabel:`Vendor` column under a specific product
+   form included in the order, the values set on the product form carry over to the |PO|, and
+   **must** be changed manually, if necessary.
+
+Once ready, create a second alternative quotation by clicking the :guilabel:`Alternatives` tab,
+followed by :guilabel:`Create Alternative`.
```

Comment:
Don't want to explain it in full here, because there's a dedicated section to it later on. As a compromise I added this note in:

.. note::
   An option to :guilabel:`Compare Product Lines` also appears. Clicking this navigates to the
   :guilabel:`Compare Order Lines` page, where alternative |RfQs| can be compared, side-by-side. For
   more information on this, refer to the :ref:`Compare Product Lines
   <purchase/manage_deals/compare-product-lines>` section.

Suggestion implemented ✅